lib/moonshot/shell.rb

```diff
@@ -1,7 +1,17 @@
 require 'thor'
+require 'retriable'
 
 # Mixin providing the Thor::Shell methods and other shell execution helpers.
 module Moonshot::Shell
+  DEFAULT_RETRY_OPTIONS = {
+    on: RuntimeError,
+    tries: 60,
+    multiplier: 1,
+    base_interval: 10,
```

Comment:
That seems reasonable to me. I will leave this comment as for sometime to see if other reviewers have an opinion on this.